CODING_GUIDELINES.md

```diff
@@ -1,7 +1,7 @@
 # Coding Guidelines
 
 ## Code Style
-Follow [Rust coding guidelines](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md).
+Follow [Rust coding guidelines](https://github.com/rust-lang/rust/tree/HEAD/src/doc/style-guide/src).
```

Comment:
I'm 100% sure if this guide is originally intended. Now we can't know what was the original link's content as it's 404 :/

Reply:
I would rather points to the on-line documentation and not source book: https://doc.rust-lang.org/style-guide/index.html

Reply:
fixed